myrestapi/index.js

```diff
@@ -0,0 +1,25 @@
+//this will your express server
+import express from "express";
+import cors from "cors";
+import books from "./books.js";
+
+const app=express();
+const PORT = 8080;
+
+//configuring cors middleware
+app.use(cors());
+
+//const direct
+
+app.get('/',(req, resp)=> {
+    resp.send("Hello Techtonica. first REST API")
```

Comment:
To be fair, this is an API call but not a REST API call. [REST is based around resources](https://www.redhat.com/en/topics/api/what-is-a-rest-api#:~:text=REST%20stands%20for%20representational%20state,by%20computer%20scientist%20Roy%20Fielding). Is there a resource here?